### PR TITLE
feat(order): add order package and module

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/order/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@daffodil/order",
+  "version": "0.0.0-PLACEHOLDER",
+  "description": "A package for interacting with platforms that provide geographic data information like States and Countries",
```
Comment (damienwebdev) [inline libs/order/package.json:4]: `A client-side Angular package for managing ecommerce orders agnostic to the ecommerce platform.`

File: libs/order/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "test": "ng test order --watch=false --browsers=ChromeHeadless",
+    "publish": "(cd ../../dist/order; npm publish --access=public)"
+  },
+  "homepage": "https://github.com/graycoreio/daffodil",
+  "peerDependencies": {
+    "@angular/core": "0.0.0-PLACEHOLDER",
+    "@daffodil/core": "0.0.0-PLACEHOLDER"
+  },
+  "devDependencies": {
+    "@daffodil/core": "0.0.0-PLACEHOLDER",
```
Comment (damienwebdev) [inline libs/order/package.json:26]: Remove the deps until necessary.

File: libs/order/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "scripts": {
+    "build": "ng build order --prod",
+    "lint": "ng lint order",
+    "test": "ng test order --watch=false --browsers=ChromeHeadless",
+    "publish": "(cd ../../dist/order; npm publish --access=public)"
+  },
+  "homepage": "https://github.com/graycoreio/daffodil",
+  "peerDependencies": {
+    "@angular/core": "0.0.0-PLACEHOLDER",
+    "@daffodil/core": "0.0.0-PLACEHOLDER"
```
Comment (damienwebdev) [inline libs/order/package.json:23]: Remove the deps until necessary.

**Comment by damienwebdev** [comment]
@griest024 can you also add an entrypoint for `@daffodil/order/drivers`?

File: libs/order/package.json (hunk 1 above)
Comment (griest024) [reply libs/order/package.json]: changedc

File: libs/order/package.json (hunk 2 above)
Comment (griest024) [reply libs/order/package.json]: removed

File: libs/order/package.json (hunk 3 above)
Comment (griest024) [reply libs/order/package.json]: removed

File: libs/order/package.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "bugs": {
+    "url": "https://github.com/graycoreio/daffodil/issues"
+  },
+  "scripts": {
+    "build": "ng build order --prod",
+    "lint": "ng lint order",
+    "test": "ng test order --watch=false --browsers=ChromeHeadless",
+    "publish": "(cd ../../dist/order; npm publish --access=public)"
+  },
+  "homepage": "https://github.com/graycoreio/daffodil"
```
Comment (damienwebdev) [inline libs/order/package.json:20]: You needed the `@angular/core` dep.

Comment (damienwebdev) [reply libs/order/package.json:20]:
```
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
```

Comment (griest024) [reply libs/order/package.json]: readded, would I need to specify `@angular/common` as a dep as well?

Comment (damienwebdev) [reply libs/order/package.json]: yes

Comment (griest024) [reply libs/order/package.json]: done

**Pull request merged**